rust/sealevel/client/src/context.rs

```diff
@@ -94,3 +221,20 @@ impl<'ctx, 'rpc> TxnBuilder<'ctx, 'rpc> {
             .ok()
     }
 }
+
+// Poor man's strategy for waiting for user confirmation
+fn wait_for_user_confirmation() {
+    println!("Continue? [y/n] then press Enter");
+    let mut input = [0u8; 1];
+    std::io::stdin().read_exact(&mut input).unwrap();
+    match input[0] {
+        b'y' => {
+            println!("Continuing...");
+        }
+        b'n' => {
+            println!("Exiting with code 1...");
+            std::process::exit(1)
```

Comment:
just as a heads up, I don't think this calls the `Drop` trait implementation, but this is a very simple client and it might be fine

Reply:
Ah true, I'll just panic then lol